core/domain/draft_upgrade_services_test.py

```diff
@@ -271,9 +353,9 @@ def test_convert_states_v29_dict_to_v30_dict(self):
                     'tagged_misconception_id': None
                 }
             })]
-        self.assertEqual(
-            draft_upgrade_services.DraftUpgradeUtil._convert_states_v29_dict_to_v30_dict(  # pylint: disable=protected-access,line-too-long
-                draft_change_list)[0].to_dict(),
+        # Draft change list with schema version 30, instead of
+        # tagged_misconception_id we have tag_skill_misconception_id.
```

Comment:
The wording here is a bit loose. Perhaps "Draft change list with schema version 30. The migration from schema version 29 --> 30 changes tagged_misconception_id to tag_skill_misconception_id."